public/index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
```

Comment:
Hey coding, we have now uploaded the image, Could you update this line to:

`<link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon2.png" />`

thank you

Reply:
sure, I will change now.

Reply:
done